src/components/contextpopup/ContextPopupDirective.js

```diff
@@ -162,32 +180,52 @@ goog.require('ga_what3words_service');
               }
 
               overlay.setPosition(coord21781);
-              showPopover();
+              element.show();
+              // We use a boolean instead of  jquery .is(':visible') selector
+              // because that doesn't work with phantomJS.
+              isPopoverShown = true;
```

Comment:
I prefer to initialize such variables that are used in different places (even though they are not needed). It better establishes their scope.